### PR TITLE
paging links fix

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: card/lib/card/tasks/card/file_card_creator/abstract_file_card.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def initialize mod, name, type, codename: nil, force: false
         @type = type.to_sym
         @name = name
         @force = force
-        @codename = codename || name.to_name.key
+        @codename = codename || name.underscore.tr(" ","_")
```
Comment (ethn) [inline card/lib/card/tasks/card/file_card_creator/abstract_file_card.rb]: why isn't the key better?

Comment (xithan) [reply card/lib/card/tasks/card/file_card_creator/abstract_file_card.rb]: I mentioned the reason in the commit message: If the name is plural and you don't give an optional codename argument then you probably want the codename also be plural.

Comment (xithan) [reply card/lib/card/tasks/card/file_card_creator/abstract_file_card.rb]: "I don't want the '*' of the name in the codename I add an extra codename argument" seems more intuitive to me than "The codename should also be plural like the name. I have to repeat it in the codename argument".

Comment (ethn) [reply card/lib/card/tasks/card/file_card_creator/abstract_file_card.rb]: I'm sorry, I misunderstood the code.  (and should have read the commit message).  all sounds good.

**Pull request merged**